tools/tools.mk

```diff
@@ -120,8 +120,20 @@ endif
 
 # put NPM in the path
 PATH:=$(node_dir)$(path_separator)$(PATH)
+OPENBSD_NPM_PATH=/usr/local/bin/npm
 
 $(npm):
```

Comment:
That will work, but i'm wondering whether it would make more sense to redefine `npm` variable itself to point at `/usr/local/bin/npm` (and not provide the `$(npm): ` rule). Because it's a dependency of targets that require it, if the file is present it will just work, otherwise it will say something like `no rule to make /usr/local/bin/npm`. WDYT?

Should we do the same thing for `freebsd` (not sure how compatible those are)?

Reply:
You are right, this will be better. I will bake another diff.
I don't know about FreeBSD, I might check later.